### ISSUE TITLE
Blockbase & Child themes - WooCommerce quantity field is too small 

### ISSUE THREAD

**Issue by mikeicode** [Automattic/themes#5761]
### Quick summary

The quantity field isn't long enough to be visible/usable.

### Steps to reproduce

Look at the quantity field on the following:

* single product page
* cart
* cart on mobile

### What you expected to happen

The quantity field should be long enough to display the numbers, like this Varia example:

<img width="854" alt="Screen Shot 2021-12-08 at 11 42 11" src="https://user-images.githubusercontent.com/10353785/145253091-18a6869b-c073-465b-909c-045afda3ab39.png">

### What actually happened

Single product:

<img width="747" alt="Screen Shot 2021-12-08 at 11 41 05" src="https://user-images.githubusercontent.com/10353785/145253183-75447bd0-10fa-4ae9-b285-b3c935b1a987.png">

Cart on desktop:

<img width="696" alt="Screen Shot 2021-12-08 at 11 45 59" src="https://user-images.githubusercontent.com/10353785/145253208-9bca8bc3-a04d-4452-8a87-e7eb3706dd42.png">

Cart on mobile:

<img width="421" alt="Screen Shot 2021-12-08 at 11 46 34" src="https://user-images.githubusercontent.com/10353785/145253284-d23b9a63-3964-426a-aa15-8e06b61f2aed.png">

### Context

I'm testing WooCommerce on Blockbase to find styling issues that may result in support requests (especially with Woo on Simple).

### Simple, Atomic or both?

Atomic

### Theme-specific issue?

Blockbase & Child themes

### Browser, operating system and other notes

Firefox, Chrome

### Reproducibility

Consistent

### Severity

Some (< 50%)

### Available workarounds?

Yes, easy to implement

### Workaround details

Could use custom CSS but that's not ideal.

**Comment by alshakero** [comment]
Hi @inaikem! Long time 👋🏼 

IDK tbh! I'll mention @Automattic/theam hoping they'd pick it up. I don't mind merging, it's a tiny tiny PR, but I have no idea how to deploy.

**Comment by MaggieCabrera** [comment]
> Hi @inaikem! Long time 👋🏼
> 
> IDK tbh! I'll mention @Automattic/theam hoping they'd pick it up. I don't mind merging, it's a tiny tiny PR, but I have no idea how to deploy.

Hi! thanks for the PR, I think we missed it. I added it to our board for review. Once it's approved, the instructions to deploy are on the readme file on this repo, let us know if you have any issues with it! Not everyone has permissions to deploy on the themes repo on dotcom but our deploy tools should do 95% of the work for you and give you a diff patch for us to deploy

**Comment by MaggieCabrera** [comment]
This was fixed via https://github.com/Automattic/themes/pull/5354

**Comment by Gustavo-Hilario** [comment]
We got another report here: 5170209-zen. I could reproduce it on a test site.

**Theme: Kerr (blockbase child theme)**

I'm reopening this issue. Hopefully, someone can take another look into this.

**Comment by edwinho89** [comment]
35639630-hc

Theme: Payton

The issue can be reproduced easily on desktop view, while on mobile it looks as expected.

**Comment by cbravobernal** [comment]
I could not reproduce it in both themes. Could have been fixed [here](https://github.com/Automattic/themes/pull/5354)?

![Image](https://github.com/user-attachments/assets/cf940394-6e3f-456d-81b8-67b3eec936c5)
![Image](https://github.com/user-attachments/assets/163ed155-318f-462d-8e31-e3e3f8499861)
![Image](https://github.com/user-attachments/assets/0427717a-6874-4057-a928-7e6ad3605636)
![Image](https://github.com/user-attachments/assets/c4cfcd75-eaa7-404f-9676-f40dbe44e5d9)

**Comment by mikachan** [comment]
> Could have been fixed https://github.com/Automattic/themes/pull/5354?

Yeah, that sounds like it's probably the case, so I think it's fine to close this issue. We can always re-open if it happens again.
